0-ad-history-mod-package/simulation/templates/gaia/fauna_bear_brown.xml

```diff
@@ -26,7 +26,7 @@
     <Icon>gaia/fauna_bear_brown.png</Icon>
     <History>
         The Brown Bear has the widest distribution of the bear species, and it ranges from North America to Europe to Asia. Due to its widespread habitat, there are many subspecies of Urs Arctos, some of which look quite different. 
-	Brown bears have dense fur coats that can not only be brown but also blond or black. Brown bears are best recognized by the large hump on their shoulders. They can grow up to nine feet long and can weigh 550 pounds. While it usually eats plants, a Brown Bear would not hesitate to eat meat, and of all the species of bear the brown bear is probably the most qualified to take it due to its size and power. Brown Bears hibernate during the Winter to avoid the food shortages.
+	      Brown bears have dense fur coats that can not only be brown but also blond or black. Brown bears are best recognized by the large hump on their shoulders. They can grow up to nine feet (2.74 meters) long and can weigh 550 pounds(550 kilograms). While it usually eats plants, a Brown Bear would not hesitate to eat meat, and of all the species of bear the brown bear is probably the most qualified to take it due to its size and power. Brown Bears hibernate during the Winter to avoid the food shortages.
```

Comment:
```suggestion
	      Brown bears have dense fur coats that can not only be brown but also blond or black. Brown bears are best recognized by the large hump on their shoulders. They can grow up to nine feet (2.74 meters) long and can weigh 550 pounds (550 kilograms). While it usually eats plants, a Brown Bear would not hesitate to eat meat, and of all the species of bear the brown bear is probably the most qualified to take it due to its size and power. Brown Bears hibernate during the Winter to avoid the food shortages.
```